test/runtest_coverage.py

```diff
@@ -81,6 +105,8 @@ def main():
     
     print_non_supplied(allConfig)
 
+    # draw_graph(stacks)
```

Comment:
commented out bc of the extra depedencies, but want to check in the code anyways